task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -128,6 +126,14 @@ spec:
           exit 1
       fi
 
+      INDEX_REPOSITORY="$(echo "$IMAGE" | cut -d@ -f1 | cut -d: -f1)"
+      MANIFEST_DIGESTS=$(buildah manifest inspect "$IMAGE" | jq -e ".manifests[].digest")
+      image_manifests=""
+      for i in $MANIFEST_DIGESTS
+      do
+        image_manifests="${image_manifests} ${INDEX_REPOSITORY}@${i},"
+      done
```

Comment:
A minor suggestion is using use array and pipe to replace this for-loop. The expansion `${image_manifests:1:-1}` can be then replaced with array expansion and `tr` to construct a string with substrings separated by ",".